tests/FsCheck.Test/Gen.fs

```diff
@@ -21,7 +21,20 @@ module Gen =
         lazy (  Gen.elements l
                 |> sample 50
                 |> List.forall (isIn l))
-    
+
+    [<Property>]
+    let GrowingElements (PositiveInt i) =
+        let sizes = [ 1..i ]
```

Comment:
The `PositiveInt i` input seems superfluous, since for `i` the check is essentially exhaustive. I'd just make this a `Fact` and pick a number for `i` (say, 100).